certified-connectors/Tesseron ASM Basic Data/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1085 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Tesseron ASM Basic Data",
+    "description": "This connector allows you to use Tesseron ASM Basic Data API to create, edit and search business partners and contacts. ",
```

Comment:
@developmentLL the word `API` is not allowed. Remove it or replace with something like application or system.
Please make sure that there are no grammatical errors and everything is typographically correct.